plugins/woocommerce-admin/client/products/sections/images-section.tsx

```diff
@@ -161,43 +173,83 @@ export const ImagesSection: React.FC = () => {
 							</CardBody>
 						) : (
 							<CardBody>
-								<MediaUploader
-									onError={ () => null }
-									onSelect={ ( file ) => {
-										if (
-											images.find(
-												( img ) => file.id === img.id
-											) === undefined
-										) {
-											setValue( 'images', [
-												...images,
-												file,
-											] );
-										}
-									} }
-									onUpload={ ( files ) => {
-										if ( files[ 0 ].id ) {
-											setValue( 'images', [
-												...images,
-												...files,
-											] );
-										}
+								<FormFileUpload
+									accept="image/*"
```

Comment:
I think this might need to be updated to account for the `allowedMediaTypes` prop.

Reply:
Good idea! I changed that in the commit [aae35d0](https://github.com/woocommerce/woocommerce/pull/35358/commits/aae35d0b7acb3995d2d5249630acdb2ef6fe72e5)